PowerEditor/src/ScintillaComponent/FindReplaceDlg.h

```diff
@@ -124,7 +124,7 @@ friend class FindReplaceDlg;
 	void addSearchLine(const TCHAR *searchName);
 	void addFileNameTitle(const TCHAR * fileName);
 	void addFileHitCount(int count);
-	void addSearchHitCount(int count, int countSearched, bool isMatchLines, bool searchedEntireNotSelection);
+	void addSearchResultInfo(int count, int countSearched, bool isMatchLines, bool searchedEntireNotSelection, const FindOption *pFindOpt);

```

Comment:
I see you have added back `bool isMatchLines`. I didn't do any request to add it back. Contrarily, I think it's good to remove it, since the value of "isMatchLines" can be found in **pFindOpt**.

Reply:
>I see you have added back bool isMatchLines. I didn't do any request to add it back.

You requested it indirectly with the other changes you wanted me to put back.

---

>good to remove it, since the value of "isMatchLines" can be found in pFindOpt.

Yes, I agree.  Which was why I did it in the earlier state of the PR.

But...someone at some earlier point decided to do make this variable `true` instead of `false`:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/bf37d4714f572efa13088863ef6e8ef0132c4021/PowerEditor/src/ScintillaComponent/FindReplaceDlg.h#L85

which interferes with a main search being able to use the `pFindOpt` member (instead of the `isMatchLines` variable) without further changes.

So, several of the other changes I did in my earlier PR became necessary to compensate for this.
And you had previously wanted me to restore those.

So I'm at a quandary... now I can put _some_ of those changes back in and you won't disapprove, in order to get rid of passing `isMatchLines` as a separate parameter?

(Note that I'm NOT talking about the "line filter" output stuff -- that's been put back)

Reply:
> But...someone at some earlier point decided to do make this variable true instead of false:

You can use "YOU" instead of "someone", can't you? Otherwise, I suggest you to use "Your Majesty" :)

The reason of the member `_isMatchLineNumber` being set to true, because it's used only in "Find in Finder" operation. So it won't influence the other operations. I don't think it's necessary to change this logic. Please let me know if I'm wrong.